server/cherry_pick.go

```diff
@@ -41,10 +86,16 @@ func (s *Server) handleCherryPick(ctx context.Context, commenter, body string, p
 	if !pr.Merged.Valid || !pr.Merged.Bool {
 		return nil
 	}
-	cmdOut, err := s.doCherryPick(ctx, strings.TrimSpace(args[1]), nil, pr)
-	if err != nil {
-		msg = fmt.Sprintf("Error trying doing the automated Cherry picking. Please do this manually\n\n```\n%s\n```\n", cmdOut)
-		return err
+
+	select {
+	case s.cherryPickRequests <- &cherryPickRequest{
+		pr:      pr,
+		version: strings.TrimSpace(args[1]),
```

Comment:
I think we need to change the type of `milestone` to `*int` 

In `doCherryPick` on line 207, we check if `milestoneNumber != nil`, which is this case it would not be nil because milestone would be 0.